From edafba60cbcaab71266f53f3359126c69106b21c Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Zbigniew=20J=C4=99drzejewski-Szmek?= <zbyszek@in.waw.pl>
Date: Sun, 18 Sep 2011 11:06:07 +0200
Subject: [PATCH] systemadm: make the dependency listing  selectable

There's no reason to forbid selecting the text.
(cherry picked from commit 37d3b881f1cc7c18d23013ff29429ab67ab5b61d)
---
 src/systemadm.vala |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/systemadm.vala b/src/systemadm.vala
index a3068f1..e7fa354 100644
--- a/src/systemadm.vala
+++ b/src/systemadm.vala
@@ -238,7 +238,7 @@ public class MainWindow : Window {
                 job_type_label = new RightLabel();
 
                 unit_dependency_label.set_track_visited_links(false);
-                unit_dependency_label.set_selectable(false);
+                unit_dependency_label.set_selectable(true);
                 unit_dependency_label.activate_link.connect(on_activate_link);
 
                 unit_fragment_path_label.set_track_visited_links(false);
